### ISSUE TITLE
Doppio fails to start on Safari

### ISSUE THREAD

**Issue by jimfb** [plasma-umass/doppio#426]
On Safari, I get a stack trace that looks like this:

```
SyntaxError: Unexpected token '>'
eval
eval code
eval
(anonymous function)jvm.ts:518
evalNativeModulejvm.ts:569
(anonymous function)jvm.ts:641
(anonymous function)OverlayFS.ts:57
iutil.ts:149
```

cc @jvilk @hrj

**Comment by jvilk** [comment]
That's odd. Looks like it doesn't like something we're producing during eval.

Will take a look when I have time, hopefully this weekend. Others can feel free to beat me to the punch.

**Comment by jvilk** [comment]
I'm not getting that when I run the unit tester with `test-browser`, but test 15 seems to hang for no explicable reason.

**Comment by jvilk** [comment]
I cannot reproduce this, Jim. I just ran all of the Safari unit tests w/o a WebWorker and it worked fine. Do you have custom native methods? Maybe you have a bug in them?

**Comment by jimfb** [comment]
I dug a little deeper on this one and discovered the root cause.  I was wrong to blame doppio, sorry, closing.

Details: We were using ES6 in our native method implementation.  Firefox and chrome support ES6 natively, so it "just worked" and so we failed to realize that we weren't transforming our code before deploying.  The ES6 code failed to run in Safari, but the error pointed straight at doppio internals so I failed to notice that it was running user-land code.

Once again, sorry for the noise.
